website_sale_product_attachment/tests/test_tour.py

```diff
@@ -0,0 +1,22 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+# Copyright 2021 Tecnativa - Víctor Martínez
+
+import odoo.tests
+
+
+@odoo.tests.tagged("post_install", "-at_install")
+class TestWebsiteSaleProductAttachmentTourl(odoo.tests.HttpCase):
+    def setUp(self):
+        super().setUp()
+        attachment = self.env.ref("website.library_image_11")
+        product = self.env.ref("product.product_product_4_product_template")
+        product.website_attachment_ids = [(6, 0, [attachment.id])]
+
+    def test_tour(self):
+        tour = "website_sale_product_attachment_tour"
+        self.phantom_js(
+            "/",
+            "odoo.__DEBUG__.services['web_tour.tour'].run('%s')" % tour,
+            "odoo.__DEBUG__.services['web_tour.tour'].tours.%s.ready" % tour,
+            login="admin",
+        )
```

Comment:
You could migrate this to use https://www.tecnativa.com/web#id=28488&model=project.task&view_type=form&cids=1&menu_id=1
It simplifies the call, and also saves some work when migrating to v14, as `phantom_js` is deprecated.